apps/api/src/campaign-application/campaign-application.controller.spec.ts

```diff
@@ -2,21 +2,37 @@ import { Test, TestingModule } from '@nestjs/testing'
 import { CampaignApplicationController } from './campaign-application.controller'
 import { CampaignApplicationService } from './campaign-application.service'
 import { SpyOf, autoSpy } from '@podkrepi-bg/testing'
```

Comment:
Some unuses stuff that github(or an action?) points out - SpyOf and CamapignTypeCategroy